webapi/Lib/WebApi/MccSoft.WebApi/Middleware/ErrorHandlerMiddleware.cs

```diff
@@ -121,7 +122,8 @@ IActionResult result
     )
     {
         string body = await httpContext.Request.ReadAll();
-        LogContext.PushProperty("RequestBody", body);
+        using (LogContext.PushProperty("RequestBody", body))
```

Comment:
please understand how `using` and `PushProperty` works.
in a way it is right now it will never log RequestBody

Reply:
Fixed